Bottom: 53ca16769d700d40a9cc9ff9550f5261e5354ea0
Top:    042858a8e1a3b72db46c34a66204e8df2975744c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-24 14:31:12 -0700

rsockets: Change the default QP size from 512 to 384

Simple bandwidth tests using rstream showed no difference in
performance between using a QP sized to 384 entries versus 512.
Reduce the overhead of a default rsocket by using 384 entries.
A user can request a larger size by calling rsetsockopt.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/rsocket.c b/src/rsocket.c
index b89ef42..c111797 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -54,7 +54,7 @@
 #define RS_INLINE 64
 #define RS_OLAP_START_SIZE 2048
 #define RS_MAX_TRANSFER 65536
-#define RS_QP_SIZE 512
+#define RS_QP_SIZE 384
 #define RS_QP_MAX_SIZE 0xFFFE
 #define RS_QP_MIN_SIZE 8
 #define RS_QP_CTRL_SIZE 4
